assets/js/frontend-form.js

```diff
@@ -511,6 +511,31 @@
             WP_User_Frontend.removeErrors(self);
             WP_User_Frontend.removeErrorNotice(self);
 
+            /* ======== Validate: Phone field ======== */
```

Comment:
You're adding a new section every time for validation, that's not the idea. Initially it was added for required fields checking, now more and more logics are being added. Try to refactor this into something that can be used in all cases instead adding a new logic, like, iterate through each items in the form, and based on the type:

- Check for required fields and validate
- Check for limits
- Check for proper formatting, etc.